diff --git a/node_modules/@wdio/types/build/Capabilities.d.ts b/node_modules/@wdio/types/build/Capabilities.d.ts
index 945fb98..e17332d 100644
--- a/node_modules/@wdio/types/build/Capabilities.d.ts
+++ b/node_modules/@wdio/types/build/Capabilities.d.ts
@@ -146,7 +146,7 @@ export interface WithRequestedCapabilities {
      * })
      * ```
      */
-    capabilities: RequestedStandaloneCapabilities;
+    capabilities?: RequestedStandaloneCapabilities;
 }
 /**
  * The `capabilities` property is a required property when defining a testrunner configuration.
