diff --git a/node_modules/vite/dist/node/index.d.ts b/node_modules/vite/dist/node/index.d.ts
index 023a11a..e72d5b0 100644
--- a/node_modules/vite/dist/node/index.d.ts
+++ b/node_modules/vite/dist/node/index.d.ts
@@ -2770,7 +2770,9 @@ interface ConfigPluginContext extends Omit<MinimalPluginContext, "meta" | "envir
 }
 interface MinimalPluginContextWithoutEnvironment extends Omit<MinimalPluginContext, "environment"> {}
 declare module "rolldown" {
-  interface MinimalPluginContext extends PluginContextExtension {}
+  interface MinimalPluginContext {
+    environment: Environment;
+  }
   interface PluginContextMeta extends PluginContextMetaExtension {}
 }
 /**
