diff --git a/node_modules/webdriverio/build/index.js b/node_modules/webdriverio/build/index.js
index 49fdef4..518d80b 100644
--- a/node_modules/webdriverio/build/index.js
+++ b/node_modules/webdriverio/build/index.js
@@ -3763,7 +3763,7 @@ import { getBrowserObject as getBrowserObject6 } from "@wdio/utils";
 
 // src/utils/interception/index.ts
 import logger8 from "@wdio/logger";
-import { URLPattern } from "urlpattern-polyfill";
+import { URLPattern } from "node:url";
 
 // src/utils/Timer.ts
 var TIMEOUT_ERROR = "timeout";
diff --git a/node_modules/webdriverio/build/node.js b/node_modules/webdriverio/build/node.js
index ec3e2ae..6cd9a8e 100644
--- a/node_modules/webdriverio/build/node.js
+++ b/node_modules/webdriverio/build/node.js
@@ -3950,7 +3950,7 @@ import { getBrowserObject as getBrowserObject7 } from "@wdio/utils";
 
 // src/utils/interception/index.ts
 import logger9 from "@wdio/logger";
-import { URLPattern } from "urlpattern-polyfill";
+import { URLPattern } from "node:url";
 
 // src/utils/Timer.ts
 var TIMEOUT_ERROR = "timeout";
diff --git a/node_modules/webdriverio/build/utils/interception/index.d.ts b/node_modules/webdriverio/build/utils/interception/index.d.ts
index b757b23..78c79dd 100644
--- a/node_modules/webdriverio/build/utils/interception/index.d.ts
+++ b/node_modules/webdriverio/build/utils/interception/index.d.ts
@@ -1,6 +1,6 @@
 import type { JsonCompatible } from '@wdio/types';
 import { type local, type remote } from 'webdriver';
-import { URLPattern } from 'urlpattern-polyfill';
+import { URLPattern } from 'node:url';
 import type { MockFilterOptions, RequestWithOptions, RespondWithOptions } from './types.js';
 import type { WaitForOptions } from '../../types.js';
 type RespondBody = string | JsonCompatible | Buffer;
